infra/debian/runtime/control

```diff
@@ -6,20 +6,40 @@ Build-Depends: cmake, debhelper (>=9)
 Standards-Version: 3.9.8
 Homepage: https://github.com/Samsung/ONE
 
-Package: nnfw
+Package: onert
+Replaces: nnfw (<< 1.29.0)
+Breaks: nnfw (<< 1.29.0)
 Architecture: amd64
 Multi-Arch: same
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: one-runtime package
 
-Package: nnfw-dev
+Package: nnfw
+Architecture: all
+Priority: optional
+Section: oldlibs
+Depends: onert, ${shlibs:Depends}, ${misc:Depends}
+Description: one-runtime transitional package
+  This is a transitional package. It can safely be removed.
+
+Package: onert-dev
+Replaces: nnfw-dev (<< 1.29.0)
+Breaks: nnfw-dev (<< 1.29.0)
 Architecture: amd64
 Multi-Arch: same
-Depends: nnfw, ${shlibs:Depends}, ${misc:Depends}
+Depends: onert, ${shlibs:Depends}, ${misc:Depends}
 Description: one-runtime development package
 
-Package: nnfw-plugin-dev
```

Comment:
(Q) Is it your intention not to create `nnfw-plugin-dev`'s transitional package?

Reply:
Right. Nobody is using `nnfw-plugin-dev` now.